package/MDAnalysis/coordinates/base.py

```diff
@@ -2045,8 +2045,29 @@ def add_transformations(self, *transformations):
 
         """
 
+        # Checks if list is passed
+        if isinstance(transformations, list):
+            six.rais_from(
```

Comment:
[*args unpacking](https://realpython.com/python-kwargs-and-args/#using-the-python-args-variable-in-function-definitions) means that ``transformations`` will always be a tuple.